pkg/controller/metadata.go

```diff
@@ -0,0 +1,53 @@
+package controller
+
+import (
+	"context"
+	"regexp"
+	"strconv"
+
+	goGitlab "github.com/xanzy/go-gitlab"
+
+	"github.com/mvisonneau/gitlab-ci-pipelines-exporter/pkg/gitlab"
+)
+
+func parseVersionRegex(version string) (int, int, int, string) {
```

Comment:
what would you think of leveraging [golang.org/x/mod/semver](https://pkg.go.dev/golang.org/x/mod/semver)(golang.org/x/mod/semver) instead?

Reply:
Sure, that cleans things up.

Reply:
thanks a ton! 🙇